src/less/actionable/ds6/actionable.less

```diff
@@ -12,3 +12,11 @@
 @actionable-image-button-border-color-active: @color-text-secondary;
 
 @import "../base/actionable.less";
+
+button.icon-btn,
+a.icon-link {
+    &:focus svg,
```

Comment:
Should we have opacity for background image? So like `&:focus span`

Also I think for SVG it should we use fill instead of color?

Reply:
We'd need a new base64 for every icon to do that though, right? (see comment in description).

Is there a benefit of using fill over color?

Reply:
Yes, if the svg's `<path>` has a fill specified, then color won't override it. However fill will. 
It's not a big deal, however some SVGs with predefined fills won't get that hover color (and if we use fill in other places in the code then it will also take precedence)

Reply:
Done.